### PR TITLE
Transitive closure to main

### PR DESCRIPTION
**Pull request by sctfn**
In my effort to eliminate TrPred and t+, this PR moved t++ to main and reproves frmin

### CODE REVIEW

**Review by icecream17 — APPROVED** [review]

File: set.mm [hunk 1]
```diff
@@ -86086,6 +86142,39 @@ general ordinal versions of these theorems (in this case ~ oa0r ) so
     WFGWSVTWGVTWGIDWFVSGUDZWTPWSUEPUGWSVQWKWDDWFVLUHUFTVEVLVSWFAMERSBMERSCMERSD
     MERSUIUJ $.
 
+  ${
+    $d A x y $.  $d B x y $.
+    $( Finite ordinal subtraction cancels on the left.  (Contributed by Scott
+       Fenton, 17-Oct-2024.) $)
+    nnasmo $p |- ( A e. _om -> E* x e. _om ( A +o x ) = B ) $=
```
[the remark this answers is not in the document]
Comment (sctfn) [reply set.mm]: Done

**Comment by avekens** [comment]
> In my effort to eliminate TrPred and t+, this PR moved t++ to main and reproves frmin

I did not follow this activity (intensively enough) in the past, so I currently see no benefit to introduce a new definition in main. The proof of ~frmin becomes (a lot) longer. Maybe you can create an issue for this, and provide an outline there about the objectives and how you want to proceed. And that the definitions TrPred and t+will be eliminated at the end (entirely replaced by t++?).

**Comment by sctfn** [comment]
Issue opened in #4456

File: changes-set.txt [hunk 2]
```diff
@@ -93,6 +93,14 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
+27-Nov-24 eldifsucnn [same]      moved from SF's mathbox to main set.mm
```
Comment (avekens) [inline changes-set.txt:96]: Indentation: please revove one space in front of "moved from"

**Review by avekens — APPROVED** [review]

**Comment by sctfn** [comment]
@icecream17 does this plan make sense? If so, can I get an approval?

**Review by icecream17 — APPROVED** [review]

**Pull request merged**